dtfs-central-api/src/services/state-machines/fee-record/event-handlers/helpers/calculate-fixed-fee.test.ts

```diff
@@ -16,59 +13,30 @@ describe('calculateFixedFeeAdjustment', () => {
-      // Arrange
-      const utilisation = 100000;
-      const interestPercentage = 5;
-      const dayCountBasis = 365;
+  it('returns the product of the utilisation, interest percentage (divided by 100), bank admin fee percentage (fixed at 0.9) and number of days remaining in the cover period divided by the day count basis and rounds to 2 decimal places', () => {
+    // Arrange
+    const utilisation = 100000;
+    const interestPercentage = 5;
+    const dayCountBasis = 365;
 
-      const reportPeriod = getReportPeriodForDate(TODAY);
-      const coverStartDateAfterReportPeriod = addMonths(TODAY, 1);
+    const aDate = new Date(2024, 0, 1);
 
-      // Number of days remaining = 368
-      const coverEndDate = addDays(coverStartDateAfterReportPeriod, 368);
+    const reportPeriod = getReportPeriodForDate(aDate);
 
-      // Act
-      const result = calculateFixedFee({
-        utilisation,
-        interestPercentage,
-        dayCountBasis,
-        reportPeriod,
-        coverStartDate: coverStartDateAfterReportPeriod,
-        coverEndDate,
-      });
+    // Number of days of cover from end of report period to cover end date
+    // is days between 1st Feb 2024 and 1st Sep 2024 = 213
+    const coverEndDate = new Date(2024, 8, 1);
 
-      // Assert
-      expect(result).toBe(4536.99); // 100000 * (5 / 100) * 0.9 * 368 / 365 = 4,536.98630... = 4,536.99 (ROUNDED)
+    // Act
+    const result = calculateFixedFee({
+      utilisation,
+      interestPercentage,
+      dayCountBasis,
+      reportPeriod,
+      coverEndDate,
     });
-  });
-
-  describe('when the report period starts after the cover start date', () => {
-    it('returns the product of the utilisation, interest percentage (divided by 100), bank admin fee percentage (fixed at 0.9) and number of days remaining in the cover period divided by the day count basis and rounds to 2 decimal places', () => {
-      // Arrange
-      const utilisation = 100000;
-      const interestPercentage = 5;
-      const dayCountBasis = 365;
-
-      const reportPeriod = getReportPeriodForDate(TODAY);
-      const coverStartDateBeforeReportPeriod = subMonths(TODAY, 1);
 
-      // Number of days remaining = 432
-      const coverEndDate = addDays(startOfMonth(TODAY), 432);
-
-      // Act
-      const result = calculateFixedFee({
-        utilisation,
-        interestPercentage,
-        dayCountBasis,
-        reportPeriod,
-        coverStartDate: coverStartDateBeforeReportPeriod,
-        coverEndDate,
-      });
-
-      // Assert
-      expect(result).toBe(5326.03); // 100000 * (5 / 100) * 0.9 * 432 / 365 = 5,326.02739... = 5,326.03 (ROUNDED)
-    });
+    // Assert
+    expect(result).toBe(2626.03); // 100000 * (5 / 100) * 0.9 * 213 / 365 = 2,626.0274 = 2,626.03 (ROUNDED)
```

Comment:
Can we use `toEqual` throughout please

Reply:
good shout, at some point we should just get rid of all the toBes remaining in one fel sweep, I like the idea of no overhead in worrying between toBe and toEqual!

Reply:
It was already done 😎  https://github.com/UK-Export-Finance/dtfs2/pull/3654

Reply:
oh sweet! nice :D